packages/admithub-lightbox/package.js

```diff
@@ -8,12 +8,17 @@ Package.onUse(function(api) {
   api.use([
     'accounts-base',
     'stylus',
-    'telescope:core@0.24.0',
+    'telescope:core',
```

Comment:
FYI: telescope dropped a new version today which has breaking changes for us. Please don't drop version pinning.